templates/quintupleo.yaml

```diff
@@ -137,7 +137,11 @@ resources:
     type: OS::Neutron::Subnet
     properties:
       network: {get_resource: provision_network}
-      name: provision-subnet
```

Comment:
Oh, oops.  The pattern with all of the rest of the subnets is actually to just name them the same as the network.  Since Neutron already separates them it shouldn't be a problem to confuse a  network for a subnet.